libs/ui/src/lib/notice/notice.component.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+} from '@angular/core';
+import { CviIconName } from '../../../../../dist/libs/icons';
+import { NoticeSeverity } from './notice';
+import { NoticeSeverityToHeaderIconPipe } from './notice-severity-to-header-icon.pipe';
+
+@Component({
+  selector: 'cvi-ng-notice',
+  templateUrl: './notice.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NoticeComponent implements AfterViewInit {
+  @Input() severity: NoticeSeverity = 'info';
+  @Input() iconName?: CviIconName;
+  constructor(private readonly iconPipe: NoticeSeverityToHeaderIconPipe) {}
+  ngAfterViewInit(): void {
```

Comment:
This basically throws the error every time the component is rendered. Do we really need right now to implement the method then?